### ISSUE TITLE
CLI doesn't report error status code returned from the CC

### ISSUE THREAD

**Issue by jvshahid** [cloudfoundry/cli#1136]
## Command  

`cf status`

## What occurred

cli displayed 

```
Getting spaces in org system as admin...

name
No spaces found
```

## What you expected to occur

Expected the cli to say that the CC returned a 500 status code. Below is the same command with verbose logging:

```
$ cf -v spaces
Getting spaces in org system as admin...


REQUEST: [2017-05-09T18:05:16-04:00]
GET /v2/organizations/ab00d2eb-b5f3-4448-a31b-91ced5a73309/spaces?order-by=name&inline-relations-depth=1 HTTP/1.1
Host: .....
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Connection: close
Content-Type: application/json
User-Agent: go-cli 6.26.0+9c9a261.2017-04-06 / linux



RESPONSE: [2017-05-09T18:05:16-04:00]
HTTP/1.1 500 Internal Server Error
Connection: close
Content-Length: 99
Content-Type: application/json;charset=utf-8
Date: Tue, 09 May 2017 22:05:17 GMT
Server: nginx
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: 3fc485f6-5161-4ef3-4e37-9c6e88802c3b
X-Vcap-Request-Id: 3fc485f6-5161-4ef3-4e37-9c6e88802c3b::84ff9e29-8e68-4fb4-91f2-275abd90978b

{
  "error_code": "UnknownError",
  "description": "An unknown error occurred.",
  "code": 10001
}

name
No spaces found
```

## CLI Version  

```
cf version 6.26.0+9c9a261.2017-04-06
```

## CC API Endpoint Version  

```
api version:    2.75.0
```

## CF Trace

Attached above

## Platform & Shell Details  

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: LinuxMint
Description:    Linux Mint 18.1 Serena
Release:        18.1
Codename:       serena

$ uname -a
Linux pivotal-virtual-machine 4.4.0-53-generic #74-Ubuntu SMP Fri Dec 2 15:59:10 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ echo $SHELL
/bin/bash
```

## Any other relevant information  

None

**Comment by dkoper** [comment]
Thanks! We'll take a look.

**Comment by dkoper** [comment]
We have addressed this in the 6.32.0 release.
